socialhome/templates/base.html

```diff
@@ -50,9 +50,11 @@
                               <a class="nav-link" href="{% url 'account_logout' %}">{% trans "Logout" %}</a>
                           </li>
                       {% else %}
+                          {% if is_open_for_signup %}
```

Comment:
> Please test things locally before submitting a PR ;)

It's an habit I got from contributing to diaspora. I like to open early PRs everything I write is public and I can get some advice. But you're right: I should have made clear that it's WIP.